src/platform/plugins/shared/content_management/common/rpc/bulk_get.ts

```diff
@@ -14,7 +14,7 @@ import { GetResult, getResultSchema } from './get';
 
 import type { ProcedureSchemas } from './types';
 
-export const bulkGetSchemas: ProcedureSchemas = {
+export const bulkGetSchemas = {
   in: schema.object(
     {
       contentTypeId: schema.string(),
```

Comment:
In this case it may sometimes be useful to have access to the true type here. When `ProcedureSchemas` is defined with the generics lock-in...

```ts
export interface ProcedureSchemas {
  in: Type<any> | false;
  out?: Type<any> | false;
}
```

The type of `bulkGetSchemas.in` is always `Type<any> | false` and we do not get to see the actual schema. Using `satisfies ProcedureSchemas` we then get the inner `Type` while still ensuring the type conforms to `ProcedureSchemas`